dev/envs/ci-preview/.envrc

```diff
@@ -1,36 +1,19 @@
 export GENESIS_UTXO="d334afbb10b16d9b8bc1537d1da0d7c1b6b46ac71666a3a38161a39a0223db01#1"
 
-# below exports parse config/addresses.json file, which needs to regenerated for each new sidechain by running
-# sidechain-main-cli addresses ... > config/addresses.json
 export COMMITTEE_CANDIDATE_ADDRESS=$(jq -r '.addresses.CommitteeCandidateValidator' dev/envs/ci-preview/addresses.json)
 export D_PARAMETER_POLICY_ID=$(jq -r '.policyIds.DParameter' dev/envs/ci-preview/addresses.json)
 export PERMISSIONED_CANDIDATES_POLICY_ID=$(jq -r '.policyIds.PermissionedCandidates' dev/envs/ci-preview/addresses.json)
+export ILLIQUID_SUPPLY_VALIDATOR_ADDRESS=$(jq -r '.addresses.IlliquidCirculationSupplyValidator' dev/envs/ci-preview/addresses.json)
+export GOVERNED_MAP_VALIDATOR_ADDRESS=$(jq -r '.addresses.GovernedMapValidator' dev/envs/ci-preview/addresses.json)
+export GOVERNED_MAP_POLICY_ID=$(jq -r '.policyIds.GovernedMap' dev/envs/ci-preview/addresses.json)
 
 # native token observability
 export NATIVE_TOKEN_POLICY_ID='ada83ddd029614381f00e28de0922ab0dec6983ea9dd29ae20eef9b4'
 export NATIVE_TOKEN_ASSET_NAME='5043546f6b656e44656d6f'
-export ILLIQUID_SUPPLY_VALIDATOR_ADDRESS='addr_test1wrhvtvx3f0g9wv9rx8kfqc60jva3e07nqujk2cspekv4mqs9rjdvz'
```

Comment:
I have no idea why this one is different to one from addresses. We'll see if it requires ci-preview PC restart.